modules/enabling-scanner-v4-after-operator-installation.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-operator-installation_{context}"]
+= Enabling RHACS Scanner V4 after Operator installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default. If you did not enable it during installation, you can enable it after installation.
+
+First, you must enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. In the console, select *Operators* -> *Installed Operators* and select the {product-title-short} Operator.
+. Click *Central* in the menu bar.
+. Click *stackrox-central-services* to view the *stackrox-central-services* page, if you installed {product-title-short} using the default values.
+. Click the name of the cluster where Central was installed. The default value is *stackrox-central-services*.
+. Select the YAML tab.
```

Comment:
Just checking, is "YAML" a UI element? If so, maybe it should also be bold? I have trouble with this UI description stuff 😅
```suggestion
. Select the *YAML* tab.
```